docker/metadata.json

```diff
@@ -1117,6 +1117,22 @@
             "_schemaURL": "https://openlineage.io/spec/facets/1-0-0/DatasourceDatasetFacet.json",
             "name": "food_delivery_db",
             "uri": "postgres://food_delivery:food_delivery@postgres:5432/food_delivery"
+          },
+          "dataQualityAssertions": {
+            "_producer": "https://some.producer.com/version/1.0",
```

Comment:
The `_producer` in [`docker/metadata.json`](https://github.com/MarquezProject/marquez/blob/main/docker/metadata.json)  can be set to:

```
https://github.com/MarquezProject/marquez/blob/main/docker/metadata.json
```